src/openforms/authentication/tests/test_admin.py

```diff
@@ -51,3 +51,49 @@ def test_validate_invalid_kvk(self):
         self.assertContains(
             form_response, "KvK-nummer moet 8 tekens lang zijn.", html=True
         )
+
+
+class RegistratorInfoAdminValidationTest(WebTest):
+    def setUp(self):
+        super().setUp()
+        self.user = UserFactory.create(is_superuser=True, is_staff=True, app=self.app)
```

Comment:
```py
@classmethod
def setUpTestData(cls):
    ...
```

is a bit more performant for this as it only runs once for the entire testcase. Additionally, we can drop the `app=self.app` requirement if we slap the `@disable_2fa` decorator on the test.